### ISSUE TITLE
password incorrect error

### ISSUE THREAD

**Issue by boa76** [lightninglabs/lightning-terminal#655]
hi i am getting password incorrect error ,where i am using default password. how to reset the password ?
Thanks
<img width="1470" alt="Screenshot 2023-10-16 at 8 57 24 PM" src="https://github.com/lightninglabs/lightning-terminal/assets/42274418/fea7b977-2787-4480-b113-fba88a3a54fb">

**Comment by guggero** [comment]
This could be due to Lightning Terminal not being fully started. Can you check the logs please? Might need to as Umbrel support on how to do that exactly.

**Comment by boa76** [comment]
lightning-terminal

Attaching to lightning-terminal_app_proxy_1, lightning-terminal_web_1
app_proxy_1  | Validating token: a5b156c4ec3b ...
app_proxy_1  | Validating token: a5b156c4ec3b ...
app_proxy_1  | Validating token: a5b156c4ec3b ...
app_proxy_1  | Validating token: a5b156c4ec3b ...
app_proxy_1  | Validating token: a5b156c4ec3b ...
app_proxy_1  | Validating token: a5b156c4ec3b ...
app_proxy_1  | Validating token: a5b156c4ec3b ...
app_proxy_1  | Validating token: a5b156c4ec3b ...
app_proxy_1  | Validating token: a5b156c4ec3b ...
app_proxy_1  | Validating token: a5b156c4ec3b ...
web_1        | 2023-10-16 12:45:32.867 [INF] LITD: Handling gRPC web request: /lnrpc.Lightning/GetInfo
web_1        | 2023-10-16 12:45:37.405 [INF] LITD: Handling gRPC web request: /lnrpc.Lightning/GetInfo
web_1        | 2023-10-16 12:48:47.560 [INF] LITD: Handling static file request: /
web_1        | 2023-10-16 12:49:03.773 [INF] LITD: Handling gRPC web request: /lnrpc.Lightning/GetInfo
web_1        | 2023-10-16 12:49:06.497 [INF] LITD: Handling gRPC web request: /lnrpc.Lightning/GetInfo
web_1        | 2023-10-16 12:49:29.420 [INF] LITD: Handling gRPC web request: /lnrpc.Lightning/GetInfo
web_1        | 2023-10-16 12:52:06.628 [INF] LITD: Handling static file request: /
web_1        | 2023-10-16 12:55:21.113 [INF] LITD: Handling static file request: /
web_1        | 2023-10-16 12:55:24.577 [INF] LITD: Handling gRPC web request: /lnrpc.Lightning/GetInfo
web_1        | 2023-10-16 12:56:50.773 [INF] LITD: Handling gRPC web request: /lnrpc.Lightning/GetInfo

found this in umbrel node logs

**Comment by levmi** [comment]
Did you recently upgrade either the Lightning Terminal app or the Lightning Node app? Is the Lightning Node up yet? Sometimes this error occurs when the Lightning Node hasn't completed it's boot up either due to still syncing to chain or something else.

**Comment by boa76** [comment]
accidently my node has rebooted , after that only lightning terminal app unable to login issue

**Comment by boa76** [comment]
lightning node is up and running , and other apps are up & running , only Lightning terminal is having issue

**Comment by boa76** [comment]
> That's only the last couple of lines, we would need more of the `web_1` container. Did you try restarting (NOT uninstalling/re-installing) Lightning Terminal?
yes my node got restarted unexpectedly

**Comment by levmi** [comment]
How long has it been since the node restarted? Do you know how to pull the more verbose logs for Lightning Terminal so we can help you debug the issue?

**Comment by boa76** [comment]
4 hours

**Comment by boa76** [comment]
Do you know how to pull the more verbose logs for Lightning Terminal so we can help you debug the issue? can you guide me on this

**Comment by levmi** [comment]
The Umbrel architecture is particular to their setup. It might take us some time to be able to properly guide you. When these issues come up, I point people to their community chat here: https://t.me/getumbrel to get guidance directly from their team. They should be able to help you get those logs and then you can post them here and we can help from there :)

**Comment by boa76** [comment]
sure will do that ,thanks much

**Comment by its-mr-gray** [comment]
@boa76 any update on this issue? i'm facing the same problem.

**Comment by guggero** [comment]
@its-mr-gray as mentioned above, it's very likely not an issue with the password itself but the server not being fully started. Please check the log for errors.

**Comment by guggero** [comment]
A restart can help in certain situations, yes. But without knowing what the error is that causes the situation, we can only guess.

**Comment by nonfungible-human** [comment]
I had this problem over the weekend too. I had forgotten to restart my node, but it told me my password was incorrect. I did some tests and discovered that in many cases, a more accurate error message could be reported.

The function `login()` calls `validate()` which throws an exception if something goes wrong.

```
try {
  // validate the the credentials are correct
  await this.validate();
} catch (error) {
  // clear the credentials if incorrect
  this.setCredentials('');
  this._store.log.error('incorrect credentials');
  throw new Error(l('invalidPassErr'));
}
```

This is where it assumes the problem has to be an incorrect password. The exception is caught but ignored. It contains useful messages that could help identify what the problem might be.

When the node is not running, the message is:
`lnd is not running: could not set up LND clients: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing: dial tcp 127.0.0.1:10009: connect: connection refused"`

When the node is locked, the message is:
`lnd is not running: could not set up LND clients: rpc error: code = Unknown desc = wallet locked, unlock it to enable full RPC access`

When the wrong password is entered, the message is:
`Error: expected 1 macaroon, got 0`

If these error messages can be relied on, they could be used to provide more accurate messages to the web UI in some cases. More accurate error messages would make it easier for users to diagnose and fix many of these problems on their own.

**Comment by boa76** [comment]
i tired a full restart, doesnt help. @its-mr-gray @guggero

**Comment by guggero** [comment]
Can you upload the full log of Lightning Terminal please? The Umbrel support telegram group will help you find out how to get that log.

**Comment by arnjr1986** [comment]
The same error happened to me. I resolved it by stopping the umbrel and then restarting the computer. Then return normally.

**Comment by levmi** [comment]
Closing as it should be fixed with the merge of #678
